Runtime/Scripts/Layers/BinaryMeshLayer/BinaryMeshLayer.cs

```diff
@@ -31,6 +30,9 @@ public class BinaryMeshLayer : Layer
         private Mesh mesh;
         private MeshRenderer meshRenderer;
 
+        public bool createMeshcollider = false;
```

Comment:
Moet deze nog public zijn?

Reply:
Voor backwards compatiblity zo gelaten, nu wel obsolete er bijgezet, en verwezen naar de nieuwe Property. Dan kunnen we m' een latere release evt. weghalen.